### DISCUSSION TITLE
Bloodhound and PurpleKnight Signatures in Security Onion and Emerging Threats - ProofPoint

### DISCUSSION THREAD

**Discussion by smallboy69** [Security-Onion-Solutions/securityonion#11837] [General] [answered]
hi Community

hope this message finds you well

can please advise if the above Scanners have had their signatures updated in the rules sets of Security Onion Rules updates such that Emerging Threats or Proof Point latest releases

Thanks

**Comment by dougburks** [comment, marked as answer]
You can find the Emerging Threats ruleset here:
https://rules.emergingthreats.net/

If you have questions about the Emerging Threats ruleset, the best place to ask would be:
https://community.emergingthreats.net/c/rule-sigs/11

Looks like the Emerging Threats team answered a similar question here:
https://community.emergingthreats.net/t/is-bloodhound-signature-included-in-the-latest-weekly-update/1138/2

**Comment by smallboy69** [comment]
Thanks Dougburks 

Dually Noted  and appreciated 

Regards..
